src/install.rst

```diff
@@ -6,361 +6,320 @@ Installing Nextstrain
 
-                  .. note::
+   Whenever you open a new terminal window to work on a Nextstrain analysis, remember to activate the Conda environment with ``conda activate nextstrain``.
 
-                     See `Docker's installation documentation <https://docs.docker.com/engine/install/ubuntu/>`__ for alternative installation methods.
+.. admonition:: For WSL on Windows installs
+   :class: hint
 
-               2. Add your user to the `docker` group:
+   By default, your Windows home directory will not be directly accessible under your WSL home directory. When run in a WSL prompt, the following command fixes that by creating a symlink to it in your WSL home directory. This allows you to use Windows-based text editors and Linux commands all on the same files.
 
-                  .. code-block:: bash
+   .. code-block:: bash
 
-                     sudo gpasswd --add $USER docker
+         ln -ws "$(wslpath "$(wslvar USERPROFILE)")" ~/windows_home
 
-               3. Restart your machine.
-               4. Create a Conda environment named ``nextstrain``:
+   Optionally, you can customize the ``windows_home`` folder name or only link to a specific directory under your windows user (e.g. ``ln -ws "$(wslpath "$(wslvar USERPROFILE)")/Documents" ~/windows_documents``).
 
-                  .. include:: snippets/conda-create-bash.rst
+   If the command does not work, you may have to first run ``sudo apt install wslu``.
 
-               5. Install the Nextstrain CLI:
+Update an existing installation
+================================
 
-                  .. code-block:: bash
+.. tabs::
 
-                     mamba install --yes nextstrain-cli
+   .. group-tab:: Docker
 
-            .. group-tab:: Native
+      Download the latest image with the Nextstrain CLI.
 
-               .. note:: Steps for other Linux distributions (Debian, CentOS, RHEL, etc.) should be identical or very similar.
+      .. code-block:: bash
 
-               1. Create a Conda environment named ``nextstrain``:
+         nextstrain update
 
-                  .. include:: snippets/conda-create-bash.rst
+      If the output notes that an update of the Nextstrain CLI itself is available, run the suggested command (after optionally reviewing the release notes).
 
-               2. Install all the necessary software:
 
-                  .. include:: snippets/conda-install-full-bash.rst
+   .. group-tab:: Native
 
-      4. Confirm that the installation worked.
+      Update the ``nextstrain`` Conda environment.
 
-         .. code-block:: bash
+      .. code-block:: bash
 
-            nextstrain check-setup --set-default
+         mamba update -n base conda mamba
+         conda activate nextstrain
+         mamba update --all
 
-         The final output from the last command should look like this, where ``<option>`` is the option chosen in the previous step:
 
-         .. code-block:: none
+Troubleshoot a broken installation
+==================================
```

Comment:
This whole section is Conda-specific, so maybe preface it for "native" runtime only. Or, add tabs to be consistent with other sections on this page, and add troubleshooting steps for standalone CLI / Docker runtime (if any). Some that come to mind:

- Run `nextstrain check-setup`
    - Check that `nextstrain-cli is up to date`
    - Check that there are no warnings for the Docker runtime

Reply:
Good point. I added tabs in 0a560ef.